test/util/operationValidationUtils.test.js

```diff
@@ -388,21 +388,11 @@ describe('validateCareGapsParams', () => {
     }
   });
 
-  test('validateCareGapsParams throws error with both program and measure identification', async () => {
+  test('validateCareGapsParams does not throw an error with both program and measure identification', async () => {
     const INVALID_REQ = {
```

Comment:
Super picky here, but this req is no longer invalid, so should probably be renamed